ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -238,8 +251,15 @@ def create_storage_cluster(self):
         Review and Create storage cluster
 
         """
-        logger.info("Create on Review and create page")
-        self.do_click(locator=self.dep_loc["create_on_review"], enable_screenshot=True)
+        logger.info("Create storage cluster on 'Review and create' page")
+        if self.operator is OCS_OPERATOR:
+            self.do_click(
+                locator=self.dep_loc["create_on_review"], enable_screenshot=True
+            )
+        elif self.operator is ODF_OPERATOR:
+            self.do_click(
+                locator=self.dep_loc["create_storage_system"], enable_screenshot=True
+            )
         logger.info("Sleep 10 second after click on 'create storage cluster'")
         time.sleep(10)
```

Comment:
```suggestion
        time.sleep(10)
```
Adding 10 second sleep is not recommended, please utilize this function wait_until_expected_text_is_found from base_ui.py file